samples/client/petstore/php/SwaggerClient-php/lib/Model/MixedPropertiesAndAdditionalPropertiesClass.php

```diff
@@ -181,6 +181,11 @@ public function getUuid()
      */
     public function setUuid($uuid)
     {
+        if (is_null($uuid)) {
+            $this->container['uuid'] = $uuid;
+
+            return $this;
+        }
         $this->container['uuid'] = $uuid;
```

Comment:
@xaviapa seems like we've done that already: https://github.com/xaviapa/swagger-codegen/blob/23636ee245817b139e8fe16b2c1c9995aee22dab/modules/swagger-codegen/src/main/resources/php/model_generic.mustache#L132-L175

Are you using the latest master to generate the PHP API client?